### DISCUSSION TITLE
API Access to ChangeLog

### DISCUSSION THREAD

**Discussion by MrPaulAR** [netbox-community/netbox#6133] [Q&A] [answered]
Hoping I'm just missing this but is there a mechanism to query the changelog via the API?

My goal is to utilize tags and if a tag exists for over 24hr then remove that tag.  Since this data exists already in the ChangeLog it's quite appealing to me if it's accessible.

**Comment by jeremystretch** [comment, marked as answer]
Yep, the changelog is available at `/api/extras/object-changes/`.
